src/zope/interface/declarations.py

```diff
@@ -436,8 +436,7 @@ def __call__(self, ob):
 def _implements(name, interfaces, classImplements):
     # This entire approach is invalid under Py3K.  Don't even try to fix
     # the coverage for this block there. :(
-    if PYTHON3: # pragma: no cover
-        raise TypeError('Class advice impossible in Python3')
+    assert not PYTHON3, "Class advice impossible in Python 3"
```

Comment:
I'm sure. Even if we take the `assert` out and manually try to use the implementation method, we get hit with another explicit TypeError:
```pycon
Python 3.6.1 (default, Jun  1 2017, 07:41:42)
[GCC 4.2.1 Compatible Apple LLVM 8.1.0 (clang-802.0.38)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> from zope.interface.declarations import _implements, classImplements
>>> from zope.interface import Interface
>>> class Foo(object):
...   _implements('implements', Interface, classImplements)
...
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<stdin>", line 2, in Foo
  File "//.tox/py36/lib/python3.6/site-packages/zope/interface/declarations.py", line 446, in _implements
    raise TypeError(name+" can be used only from a class definition.")
TypeError: implements can be used only from a class definition.
```